### ISSUE TITLE
When adding an e-mail address, keyboard tab order is unintuitive

### ISSUE THREAD

**Issue by MarcoZehe** [mozilla/blurts-server#1151]
Steps to reproduce:
1. Log into Monitor.
2. Focus the add e-mail address text field and enter another e-mail address to monitor.
3. Press Tab.
   * Expected: Firefox should focus the "Send Link to verify e-mail address" button.
   * Actual: Focus skips past the button and the Learn More links to the "Learn how hackers operate" link.

The reason is that there is a tabindex attribute with a value > 0 on the button and link. These are in the tab order, but are the first things focused when starting to tab from the top of the document. They should be in logical visual order.

The solution is to remove the tabindex attribute. It is normally not needed for inputs and other interactive elements. And changing the logical DOM-dictated keyboard focus order should only be done with very good reasoning. The way Monitor behaves now breaks user flow and is a violation of at least one Web Content Accessibility Guidelines criterium.

*Note if testing on Mac*: In System Preferences, Keyboard, check the box that says "Tab focuses all focusable elements" before testing this. Firefox honors this system preference, will otherwise only focus input fields.
